dev_tools/nyaml2nxdl/nyaml2nxdl.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python3
+"""
+Main file of nyaml2nxdl tool.
+
+To write a definition for a instrument, experiment and/or process in nxdl.xml file from a YAML
+file which details a hierarchy of data/metadata elements. It also allows both wa
```

Comment:
"To write a definition for an instrument ... It also allows back and forth conversion between ..."

Reply:
addressed together with the suggestion from @prjemian

Reply:
I haven't seen any changes to address this comment

Reply:
DONE